### ISSUE TITLE
5.0.5 Security approval

### ISSUE THREAD

**Issue by keegancsmith** [sourcegraph/sourcegraph#52746]
Security must close this issue before releases. It is the responsibility of the Security Engineer on rotation the week the release is cut. More steps will be added in the future.

- [x] All feasible CVEs are closed.
- [x] Any open CVEs are documented in the handbook.

Part of https://github.com/sourcegraph/sourcegraph/issues/52745

cc @sourcegraph/security

**Comment by keegancsmith** [comment]
For  convenience here is a link to the rc build https://buildkite.com/sourcegraph/sourcegraph/builds?branch=v5.0.5-rc.1

**Comment by keegancsmith** [comment]
I quickly scanned the trivy output for images for services we actually control. The only new one I found was for zoekt but it doesn't affect us in practice. I created this PR to silence the scanner anyways, but to get that update into the sourcegraph build will be a lot of work. IE we can't just use the latest version of zoekt since lots of stuff has changed => risk to release. https://github.com/sourcegraph/zoekt/pull/588

I propose we document we find the CVE acceptable.
